### ISSUE TITLE
Configure custom FFmpeg options dialog: missing accessibility names

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#4956]
### Steps to reproduce

When running a windows screen reader,  eg nvda:
1. Open the export audio dialog
2. Set the format to Custom FFmpeg Export
3. Press the Open custom FFmpeg format options button. The Configure custom FFmpeg options dialog opens.
4. Tab to the list of formats and the list of codecs.

### Expected behavior

The lists have accessibility names to indicate the content type of the lists. For sighted user, there are no names, but their content type is obvious from the layout.

### Actual behavior

The lists do not have accessibility names

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Windows
